doc/tutorial/sam_file/index.md

```diff
@@ -222,7 +222,7 @@ Only use
 
 With that information do the following:
   * Filter the alignment records and only take those with a mapping quality >= 30.
-    (Take a look at the tutorial \ref sequence_file_section_fun_with_ranges for a reminder how to use views on files)
+    (Take a look at the tutorial \ref sequence_file_section_fun_with_ranges for a reminder of how to use views on files)
```

Comment:
```suggestion
    (Take a look at the tutorial \ref sequence_file_section_fun_with_ranges for a reminder about using views on files)
```

or `on`

`of` is more a reminder that there are views on files
`on`/`about` also implies that there are more information to be found